### PR TITLE
Feature: Add kaigara to OpenDAX compose, integrate Sonic

### PR DESCRIPTION
**Pull request by akulakovaa**

### CODE REVIEW

File: templates/secrets.yaml.erb [hunk 1]
```diff
@@ -0,0 +1,120 @@
+secrets:
```
Comment (vshatravenko) [inline templates/secrets.yaml.erb:1]: I can only see barong in here, has there been any progress?

**Review by vshatravenko — CHANGES_REQUESTED** [review]

File: templates/secrets.yaml.erb [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        event_api_rabbitmq_port: '5672'
+        minimum_member_level_for_deposit: '1'
+        minimum_member_level_for_withdraw: '3'
+        minimum_member_level_for_trading: '2'
+        force_secure_connection: 'false'
+      secret:
+        jwt_private_key: <%= @barong_private_key %>
+        event_api_rabbitmq_username: guest
+        event_api_rabbitmq_password: guest
+        secret_key_base: faiba2shei0Ae5gahCh4aipoh3meyaFi
```
Comment (calj) [inline templates/secrets.yaml.erb:171]: This secret must be generated

File: templates/secrets.yaml.erb [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        barong_recaptcha_secret_key: <%= @config['captcha']['secretKey'] %>
+        <%- elsif @config['captcha']['type'] == 'geetest' -%>
+        barong_geetest_id: <%= @config['captcha']['siteKey'] %>
+        barong_geetest_key: <%= @config['captcha']['secretKey'] %>
+        <%- end -%>
+        barong_storage_access_key: <%= @config['storage']['accessKey'] %>
+        barong_storage_secret_key: <%= @config['storage']['secretKey'] %>
+        barong_event_api_jwt_private_key: <%= @barong_private_key %>
+        barong_event_api_rabbitmq_password: guest
+        secret_key_base: faiba2shei0Ae5gahCh4aipoh3meyaFi
```
Comment (calj) [inline templates/secrets.yaml.erb:121]: This secret must be generated

File: templates/secrets.yaml.erb [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+        jwt_public_key: <%= @barong_public_key %>
+        influxdb_host: influxdb
+        event_api_jwt_algorithm: RS256
+        event_api_rabbitmq_port: '5672'
+        minimum_member_level_for_deposit: '1'
+        minimum_member_level_for_withdraw: '3'
+        minimum_member_level_for_trading: '2'
+        force_secure_connection: 'false'
+      secret:
+        jwt_private_key: <%= @barong_private_key %>
```
Comment (calj) [inline templates/secrets.yaml.erb:168]: here we need a specific key for peatio

File: templates/secrets.yaml.erb [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+        sentryEnabled: 'false'
+        kycSteps: [
+          'email',
+          'phone',
+          'profile',
+          'document',
+          'address',
+        ]
+      private:
+        database_host: <%=  @config['database']['host'] %>
```
Comment (calj) [inline templates/secrets.yaml.erb:34]: here we need public keys of components:
```
peatio_public_key
barong_public_key
sonic_public_key
```

File: templates/secrets.yaml.erb [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+        gaTrackerKey: ''
+        rangerReconnectPeriod: '1'
+        msAlertDisplayTime: '5000'
+        incrementalOrderBook: 'true'
+        isResizable: 'false'
+        isDraggable: 'false'
+        languages: ['en', 'ru']
+        sessionCheckInterval: '15000'
+        balancesFetchInterval: '3000'
+        captcha_type: <%= @config['captcha']['type'] %>
```
Comment (dpatsora) [inline templates/secrets.yaml.erb:23]: captcha_id: <%= ssl_helper('http') %>://<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>

File: templates/secrets.yaml.erb [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+        barong_app_name: <%= @config['app']['name'] %>
+        barong_uid_prefix: ID
+        barong_sms_content_template: "Your verification code for OpenDAX: {{code}}"
+        barong_required_docs_expire: 'false'
+        barong_domain: <%= ssl_helper('http') %>://<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>
+        <%- if @config['kyc']['provider'] == 'kycaid' -%>
+        barong_kycaid_sandbox_mode: 'false'
+        barong_kycaid_api_endpoint: <%= @config['kyc']['api_endpoint'] %>
+        <%- end -%>
+        barong_api_cors_origins: https://2-6-stable.openfinex.io
```
Comment (dpatsora) [inline templates/secrets.yaml.erb:75]: barong_api_cors_origins: <%= ssl_helper('http') %>://<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>

**Pull request merged**